### DISCUSSION TITLE
Early Stopping

### DISCUSSION THREAD

**Discussion by sakerlrf** [open-mmlab/mmaction2#1171] [Q&A] [answered]
Hello,

If one needs to use **early stopping** for training, to prevent overfitting, should this part be incorporated in the `hook registry` or ***how*** this could be made? Any advice!?

If so, what is the right dir to add this new script?

Thank you!

**Comment by dreamerlin** [comment, marked as answer]
You can refer to https://github.com/open-mmlab/mmaction2/issues/1142

**Comment by dreamerlin** [comment]
By the way, if you find this repo help you, you can give us a star!
